catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -98,5 +102,14 @@ public void init() {
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList()),
         MAPPER::toJson);
+    delete(
+        "/forms/:id",
+        APPLICATION_JSON,
+        (req, res) -> {
+          String id = req.params(":id");
+          catalogFramework.delete(new DeleteRequestImpl(id));
```

Comment:
Should we try/catch here and return a 500 when the delete fails?

Reply:
Added error state to destroy call.  An error announcement is shown and new error thrown